### ISSUE TITLE
[FEATURE] Companion Support

### ISSUE THREAD

**Issue by morgurth** [NiceTSY/nice-cypher-add-ons#31]
Add the ability to create Journal Entries for Companions and then drag and drop the Type (Tyrannosaurs, Raptors, etc. from Predation for example) on to the Companion Actor sheet. Allow it to fill in things like Health, Armor, Combat Ability (i.e. Claws skill that does 3 points of dmg), Companion Abilities and their Disposition.

This is specific to Companions, which are Predation only (unfortunately), so I understand if this is too much work.

**Comment by NiceTSY** [comment]
No, it should not be. I just need to figure how to do that goodly with the new system and the v1 code :) 
Will see how I can do that.
